development/docker-compose.base.yml

```diff
@@ -22,20 +22,29 @@ services:
       db:
         condition: "service_healthy"
     <<:
-      - *nautobot-build
       - *nautobot-base
+      - *nautobot-build
   worker:
     entrypoint:
       - "sh"
       - "-c"  # this is to evaluate the $NAUTOBOT_LOG_LEVEL from the env
-      - "watchmedo auto-restart --directory './' --pattern '*.py' --recursive -- nautobot-server celery worker -l $$NAUTOBOT_LOG_LEVEL --events"  ## $$ because of docker-compose
```

Comment:
This is problematic to me, we should fix this upstream

Reply:
Could you pls. better explain why is it problematic? Any proposal?